src/error.rs

```diff
@@ -7,9 +7,11 @@ pub struct FluvioError {
     inner: FluvioInner,
 }
 
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
```

Comment:
Hmm. What does this add? I'm not against it, just unclear where we use it.

Reply:
just in order to add Display representation, so we have `debug_message` and `message`